.docker/web/run.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
May as well exit early if any step fails, right? :) 
```suggestion
#!/bin/sh -e
```